cmd/minikube/cmd/mount.go

```diff
@@ -36,42 +38,52 @@ var mountCmd = &cobra.Command{
 	Long:  `Mounts the specified directory into minikube.`,
 	Run: func(cmd *cobra.Command, args []string) {
 		if len(args) != 1 {
-			errText := `Please specify a driver name and a directory to be mounted:
-\tminikube mount MOUNT_DIRECTORY(ex:"/home")
+			errText := `Please specify the directory to be mounted: 
+\tminikube mount HOST_MOUNT_DIRECTORY:VM_MOUNT_DIRECTORY(ex:"/host-home:/vm-home")
 `
 			fmt.Fprintln(os.Stderr, errText)
 			os.Exit(1)
 		}
-		if _, err := os.Stat(args[0]); err != nil {
+		mountString := args[0]
+		idx := strings.LastIndex(mountString, ":")
+		if idx == -1 { // no ":" was present
+			errText := fmt.Sprintf(`mount directory must be in the form: 
+			HOST_MOUNT_DIRECTORY:VM_MOUNT_DIRECTORY`)
+			fmt.Fprintln(os.Stderr, errText)
+			os.Exit(1)
+		}
+		hostPath := mountString[:idx]
+		vmPath := mountString[idx+1:]
```

Comment:
I think you need to check for the cases where either hostPath or vmPath will be ""

`/c/Users:`
`:/home`
`:`